src/core/PWSprefs.cpp

```diff
@@ -210,6 +212,8 @@ const PWSprefs::stringPref PWSprefs::m_string_prefs[NumStringPrefs] = {
   {_T("AddEditFont"), _T(""), ptApplication },                      // application
   {_T("AddEditSampleText"), _T("AaBbYyZz 0O1IlL"), ptApplication},  // application
   {_T("AltNotesEditorCmdLineParms"), _T(""), ptApplication},        // application
+  {_T("lastsort"), _T("group"), ptApplication},                     // application
```

Comment:
This should be renamed to "TreeSort":
- "last" is redundant
- "lastsort" doesn't make clear what's sorted
- Capitalization is inconsistent with the other preferences.

Reply:
Replaced last sort/LastSort by TreeSort